src/Raven.Server/Commercial/LicenseHelper.cs

```diff
@@ -374,15 +370,55 @@ public void AppendResolutionSuggestions()
                         _errorBuilder.AppendLine($"- As a temporary measure, consider downgrading to the last working build ({buildInfo.FullVersion}).");
                 }
 
-                AppendSuggestionToDisableThrowOnInvalidOrMissingLicenseOption(_configuration.Licensing.ThrowOnInvalidOrMissingLicense, _isInStorageLicenseExpired);
+                AppendSuggestionToDisableThrowOnInvalidOrMissingLicenseOption(Configuration.Licensing.ThrowOnInvalidOrMissingLicense, _isInStorageLicenseExpired);
             }
 
             public void AppendGeneralSuggestions()
             {
                 _errorBuilder.AppendLine();
                 _errorBuilder.AppendLine("To resolve this issue, you may consider the following options:");
-                _errorBuilder.AppendLine("- Ensure your license key is correctly embedded in 'settings.json', set as an environment variable, or included in your 'ServerOptions' if using an embedded server or Raven.TestDriver.");
-                _errorBuilder.AppendLine("- Alternatively, check the 'License.Path' in your configuration to ensure it points to a valid 'license.json' file.");
+                _errorBuilder.AppendLine("- Ensure your license key is correctly embedded in 'settings.json' or set as an environment variable.");
+                _errorBuilder.AppendLine("- Or, check the 'License.Path' in your configuration to ensure it points to a valid 'license.json' file.");
+
+                if (Configuration.Embedded.ParentProcessId.HasValue)
+                {
+                    _errorBuilder.AppendLine("""
+                        - Or, since you are using an embedded server (TestDriver or EmbeddedServer), you can provide a valid license using one of these approaches:
+                            * For RavenTestDriver:
+                              public class YourTestClass : RavenTestDriver
+                              {
+                                  static YourTestClass()
+                                  {
+                                      ConfigureServer(new TestServerOptions
+                                      {
+                                          Licensing = new ServerOptions.LicensingOptions 
+                                          { 
+                                              License = "your license here", // Replace with your actual license,
+                                              // or
+                                              LicensePath = "path to license.json file" // Replace with the actual path to your license.json file
+                                          }
+                                      });
+                                  }
+                              }
+                              IMPORTANT: This configuration must be done in a static constructor before any server initialization.
+
+                            * For EmbeddedServer:
+                              using (var embedded = new EmbeddedServer())
```

Comment:
this is incorrect, we only support single instance, the ctor is internal, you need to use EmbeddedServer.Instance - this is what is exposed to public

Reply:
Done